### PR TITLE
Upgrade to latest V6 core

### PR DESCRIPTION
**Pull request by dvdstelt**
Connected to #47 

Upgraded to new TransportDefinition and EndpointConfiguration

### CODE REVIEW

**Comment by dvdstelt** [comment]
/cc @Particular/azure-maintainers please review

There are quite a few commits in here that could be squashed, but not sure if that's possible if they're already on remote?

File: src/Transport/Config/DefaultConfigurationValues.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿namespace NServiceBus.AzureStorageQueues
+{
+    using NServiceBus.Azure.Transports.WindowsAzureStorageQueues.Config;
+    using NServiceBus.Settings;
+
+    class DefaultConfigurationValues
+    {
+        // These are here only for the ConfigSection, can be removed in V7
```
Comment (SeanFeldman) [inline src/Transport/Config/DefaultConfigurationValues.cs]: old comment?

Comment (dvdstelt) [inline src/Transport/Config/DefaultConfigurationValues.cs]: You're right, will remove

**Comment by Scooletz** [comment]
@dvdstelt You can squash and then 'push --force' it to the remote. As long as it's a separate PR branch, you can do anything. If you need any help with this, ping me.

**Comment by SimonCropp** [comment]
the nuspec is still wrong
https://github.com/Particular/NServiceBus.AzureStorageQueues/blob/UpdateToV6/packaging/nuget/nservicebus.azurestoragequeues.nuspec#L20

needs to include the unstable of v6 eg https://github.com/Particular/NServiceBus.Callbacks/blob/develop/packaging/nuget/nservicebus.callbacks.nuspec#L22

also note there are more tokens that can be taken advantage of eg `<copyright>$copyright$</copyright>`

**Pull request merged**